src/components/forms/inputs.jsx

```diff
@@ -91,15 +91,26 @@ export const RenderCheckbox = (props) => {
   )
 };
 
-
 export const RenderRadio = (props) => {
-  const { id, input, label, wrapper, type, meta: { touched, error } } = props;
-  return (
-    <InlineError {...props} className="wfp-radio">
-        <Input {...props} />
-        <Label {...props} />
-    </InlineError>
-  )
+  const { id, input, label, wrapper, type, meta: { touched, error, checked }, hint } = props;
+  if(hint && input.checked){
```

Comment:
This is very specific behavior. as per this, a hint will be displayed only if input is checked. IMO, this logic is not generic enough to be in library. . Please add hint directly in WSS instead.